.github/workflows/test_with_tox.yml

```diff
@@ -84,7 +84,7 @@ jobs:
                     # TODO revert to latest when dropping py 3.9 and bumping matplotlib
                     os: macos-14
                     env: min
-                -   description: oldest dependencies - no plotly
```

Comment:
As `plotmin` installs `plotly`, I remove this.

Reply:
actually when re-reading our pyproject.toml plotly and matpltlib are always installed together 

https://github.com/nilearn/nilearn/blob/14a89340302aaefdad6bc41823ee60d64a13938a/pyproject.toml#L69-L75

so I wonder if we even need have a testing environement with matplotlib but not plotly

Reply:
I think it is good to keep the separate tests. I've added a new one, to only run baseline comparisons when plotly is installed. Could you check if that is OK?

Reply:
note: I am ok merging this as is and cleaning things up in another PR